app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
+
+# A method that updates the likes counter for a post.
+def update_likes_counter
+  Post.select.update(:likes_counter).count
+end
```

Comment:
- [ ] The `update_likes_counter` should be inside `Like` class
- [ ] I suggest that you use [rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` to call the `#update_likes_counter` method every time a new like has been saved to database, then make the `update_likes_counter` method private

Reply:
Done.